addons/project/static/src/components/project_task_state_selection/project_task_stage_state_selection/project_task_stage_with_state_selection.js

```diff
@@ -0,0 +1,62 @@
+import { _t } from "@web/core/l10n/translation";
+import { standardFieldProps } from "@web/views/fields/standard_field_props";
+import { Many2OneField } from "@web/views/fields/many2one/many2one_field";
+
+import { registry } from "@web/core/registry";
+import { Component } from "@odoo/owl";
+import { omit } from "@web/core/utils/objects";
+
+import { ProjectTaskStateSelection } from "../project_task_state_selection";
+
+
+export class TaskStageWithStateSelection extends Component {
```

Comment:
```suggestion
import { ProjectTaskStateSelection } from "../project_task_state_selection";

export class TaskStageWithStateSelection extends Component {
```

According to the tooling of Framework JS